registry/service.go

```diff
@@ -35,6 +36,11 @@ func (s *Service) ServiceConfig() *registry.ServiceConfig {
 	return s.config.copy()
 }
 
+// ServiceConfig returns if a host is secured
+func (s *Service) IsSecureIndex(indexName string) bool {
+	return s.config.isSecureIndex(indexName)
+}
+
```

Comment:
There's already a function you can use instead:
https://github.com/moby/moby/blob/46c5322d6a4e04ec25dc7fe01d39b9b0bfad75c1/registry/service.go#L167-L173

Reply:
OK taken in account